tests/unittests/DeviceManagerTest.cpp

```diff
@@ -549,13 +551,41 @@ TEST_P(DeviceManagerTest, ReuseModule) {
   EXPECT_TRUE(result2->isEqual(output2));
 }
 
+TEST(DeviceManagerTest, SetDeviceMemory) {
+  // Test CPU.
+  auto cpuConfigEmpty = DeviceConfig(BackendKind::CPU);
```

Comment:
can you move this cpu related stuff into `TEST(DeviceManagerTest, AvailableMemory) {` which is under ifdef GLOW_WITH_CPU.

Reply:
> can you move this cpu related stuff into `TEST(DeviceManagerTest, AvailableMemory) {` which is under ifdef GLOW_WITH_CPU.

OK. Sure